### PR TITLE
Update api key fields documentation

### PR DESCRIPTION
**Pull request by rathodmeetsatish**
## Comment Triggers

<details>
  <summary>PR Check triggers</summary>
  
- Build: `trigger build`
- Sub Category Field Check: `trigger subcategoryfieldcheck`

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -42,6 +42,6 @@ provider "harness" {
 ### Optional
 
 - `account_id` (String) The Harness account id. This can also be set using the `HARNESS_ACCOUNT_ID` environment variable.
```
Comment (rashminandasahoo) [inline docs/index.md:44]: Harness account id. You can fetch this value from the environment variable HARNESS_ACCOUNT_ID.

File: docs/index.md [hunk 2]
```diff
@@ -42,6 +42,6 @@ provider "harness" {
...
-- `api_key` (String) The Harness API key. This can also be set using the `HARNESS_API_KEY` environment variable.
+- `api_key` (String) The Harness API key. This can also be set using the `HARNESS_API_KEY` environment variable. To create an api key in firstgen follow this doc https://docs.harness.io/article/smloyragsm-api-keys#create_an_api_key
```
Comment (rashminandasahoo) [inline docs/index.md:45]: Harness API key. You can fetch this value from the environment variable HARNESS_API_KEY.
Casing -> FirstGen. 
For more information to create an API key in FirstGen, see [Create an API Key](https://docs.harness.io/article/smloyragsm-api-keys#create_an_api_key).

File: docs/index.md [hunk 3]
```diff
@@ -42,6 +42,6 @@ provider "harness" {
...
 - `endpoint` (String) The URL of the Harness API endpoint. The default is `https://app.harness.io/gateway`. This can also be set using the `HARNESS_ENDPOINT` environment variable.
```
Comment (rashminandasahoo) [inline docs/index.md:46]: URL of the Harness API endpoint. The default endpoint URL is `https://app.harness.io/gateway`. You can fetch this from the environment variable `HARNESS_ENDPOINT`

File: docs/index.md [hunk 4]
```diff
@@ -42,6 +42,6 @@ provider "harness" {
...
-- `platform_api_key` (String) The API key for the Harness next gen platform. This can also be set using the `HARNESS_PLATFORM_API_KEY` environment variable.
+- `platform_api_key` (String) The API key for the Harness next gen platform. This can also be set using the `HARNESS_PLATFORM_API_KEY` environment variable. To create an api key in nextgen follow this doc https://docs.harness.io/article/tdoad7xrh9-add-and-manage-api-keys"
```
Comment (rashminandasahoo) [inline docs/index.md:47]: API key for Harness NextGen. You can fetch this from the environment variable `HARNESS_PLATFORM_API_KEY`. For more information to create an API key in NextGen, see [Create an API Key](https://docs.harness.io/article/tdoad7xrh9-add-and-manage-api-keys).

**Review by rashminandasahoo — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 1 above)
Comment (rathodmeetsatish) [reply docs/index.md:44]: You can fetch this value from the environment variable HARNESS_ACCOUNT_ID.
this has different meaning than original text we are not trying to fetch account_id we are trying to set it similarly in other comments

Comment (rashminandasahoo) [reply docs/index.md:44]: Ok, then we can retain the original sentence (`This can also be set using the `HARNESS_ACCOUNT_ID` environment variable.`).

**Review by rashminandasahoo — APPROVED** [review]

**Pull request merged**